examples/Atlas/+atlasFrames/COMState.m

```diff
@@ -1,7 +1,10 @@
 classdef COMState < SingletonCoordinateFrame
   methods
-    function obj=COMState
-      obj = obj@SingletonCoordinateFrame('atlasFrames.COMState',6,'x');
+    function obj=COMState(dim)
```

Comment:
rename this dof instead of dim?  (as it is, you've got frame.dim = 2*dim, which is confusing)